src/pages/create.tsx

```diff
@@ -143,9 +142,6 @@ const CreateProgramPage: Page = () => {
         if (err.message.includes("duplicate key value violates unique"))
           setError(`Error: Slug "${programIdentifier}" already exists`);
         else setError("Error: " + err.message);
-      })
-      .finally(() => {
-        setLoading(false);
```

Comment:
Why is this removed? If there's an error on creating program, would it still be possible to create a program upon fixing the error?

Reply:
Really good point- I removed it because it wasn't disabling the button after the program creation was successful so perhaps including it in the .catch statement and leaving it as loading on successful creation would be the move?

Reply:
sg, including it in the catch statement is sensible to me!